examples/base_1cc/main.tf

```diff
@@ -121,6 +121,13 @@ resource "local_file" "user_data_file" {
   filename = "../user_data"
 }
 
+# Validates which Marketplace to use based on arm location
+locals {
+  is_china                         = can(regex("^china", var.arm_location))
```

Comment:
@nmizhquirizs this regex is just validating location starts with china right? We should either add case insensitivity to it. Else another terraform function like contains would work the same probably